cmd/manager/main.go

```diff
@@ -124,8 +129,39 @@ func main() {
 
 	// Start the Cmd
 	log.Info("Starting the Cmd.")
+	hadError := false
 	if err := mgr.Start(signals.SetupSignalHandler()); err != nil {
 		log.Error(err, "unable to run the manager")
+		hadError = true
+	}
+	toggle.Disable()
+
+	// Unfortunately there is no way to block until all child
+	// goroutines of the manager have finished, so sleep long
+	// enough for dangling reconciles to finish
+	time.Sleep(5 * time.Second)
```

Comment:
can we make this configurable or at least a const

Reply:
Not sure why we would make it a const? It's only used in one place. As for configurable, why would someone want to configure it?